### ISSUE TITLE
Importer names have poor contrast

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#31206]
Importer names at `/settings/import/` have poor colour contrast.

`#969CA1` + `#FFFFFF` don't offer enough contrast in this view, using the default colour theme:

<img width="741" alt="screenshot 2019-03-05 at 15 08 15" src="https://user-images.githubusercontent.com/87168/53807649-c49ff500-3f58-11e9-97e7-4bcf0960fdeb.png">

via https://webaim.org/resources/contrastchecker/?fcolor=969CA1&bcolor=FFFFFF:

![image](https://user-images.githubusercontent.com/87168/53807660-d1bce400-3f58-11e9-9f73-3f8ced241860.png)

**Comment by blowery** [comment]
I think we moved from `$gray` to `--color-neutral-light` there, but it should probably just be `--color-text` or unspecified?

**Comment by drw158** [comment]
It should be --color-text-subtle
